tests/chia_log/handlers/test_wallet_added_coin_handler.py

```diff
@@ -40,7 +47,11 @@ def testFloatPrecision(self):
         with open(self.example_logs_path / "small_values.txt", encoding="UTF-8") as f:
             logs = f.readlines()
 
-        events = self.handler.handle("".join(logs))
+        # Default mojo filter excludes the event we will expect
+        self.handler_config["min_mojos_amount"].set(0)
+        handler = WalletAddedCoinHandler(self.handler_config)
```

Comment:
Since the default used to be 0, maybe you want to do it the other way around? Keep it 0 and set it to 5 only for the test that requires to test that filter’s functionality?

Reply:
That makes sense, but I guess we need to first agree if the default should be 5 or 0 since it's a conflict between the code default and the example config.